install/helm/gloo/crds/graphql.gloo.solo.io_v1beta1_GraphQLApi.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
 spec:
   group: graphql.gloo.solo.io
   names:
+    categories:
+    - solo-io
```

Comment:
If our approach is that any enterprise CRDs will have solo-io as well, can we clarify that somewhere as an intentional decision? I can see a new CR be added, and it being challenging for a new developer to understand that expectation